PipeViewer/Form1.cs

```diff
@@ -481,7 +481,7 @@ private void openFindWindow()
 
         private void openPipeChat(string pipeName)
         {
-            PipeChat pipeChat = new PipeChat(pipeName);
+            pipeChatForm pipeChat = new pipeChatForm(pipeName);
```

Comment:
@2niknatan should be changed to `PipeChatForm`, also make sure to change the filename.